enterprise_access/apps/api/filters.py

```diff
@@ -0,0 +1,39 @@
+"""
+Filter backends for Enterprise Access API.
+"""
+
+from django.db.models import Q
+from edx_rbac import utils
+from rest_framework import filters
+
+from enterprise_access.apps.core import constants
+
+
+class IsSubsidyRequesterOrEnterpriseAdmin(filters.BaseFilterBackend):
```

Comment:
This name took me a bit to understand. Could this be named something like `FilterOnRole`, `RoleBasedFilter`, `FilterSubsidyRequestOnUserRole` (maybe too verbose)

Reply:
I might just name this SubsidyRequestFilterBackend and let the comments do the work here. Any semantic name might be too verbose here.

Reply:
cool, `SubsidyRequestFilterBackend` sounds great